recipe/meta.yaml

```diff
@@ -1,55 +1,76 @@
 {% set name = "pikepdf" %}
-{% set version = "5.6.1" %}
+{% set version = "9.4.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 03fb70ff4fada9cba864d04033f4166b5bf7fa85c90cdc05afd0ab87c9ccaa1a
+  sha256: 0108c063bc56dc2dbfc87f20533a728342a938f4c85e39773866b71255aa8388
+  patches:
```

Comment:
Please move to the **patches** directory.

Reply:
ok, done.